pytests/tests/test_baseurls.py

```diff
@@ -32,11 +32,37 @@ def teardown_test(utils):
         os.remove(filename)
 
 
-def test_multiple_baseurls(utils):
+def test_multiple_repoid(utils):
     reponame = TESTREPO
     workdir = WORKDIR
     utils.makedirs(workdir)
 
+    filename = os.path.join(utils.config['repo_path'], "yum.repos.d", REPOFILENAME)
+
+    baseurls = "http://localhost:8080/photon-test"
+    utils.create_repoconf(filename, baseurls, reponame)
```

Comment:
Does this create another repo with an id that already exists? That's not clear to me.

Reply:
Yes, This created another repo with the id def [cfg-test], which is already exist. I have removed this code in latest changes